core/controllers/cron.py

```diff
@@ -213,3 +213,12 @@ def get(self):
                     jobs.MAPPER_PARAM_MAX_START_TIME_MSEC: max_start_time_msec
                 })
             logging.warning('Deletion jobs for auxiliary entities kicked off.')
+
+        if job_models.JobModel.do_unfinished_jobs_exist(
```

Comment:
I'm a bit confused about the difference between this and the code immediately above it. Could you please explain / clarify in the code?

Reply:
The code above removes the instance of models that are sued by the MapReduce library internally. This new code removes `JobModels` i.e. the results of the jobs that we run in the past.

Reply:
Why do both have the same ` if job_models.JobModel.do_unfinished_jobs_exist(` clause? Can this code be simplified?

Reply:
It is `job_models.JobModel.do_unfinished_jobs_exist(cron_services.JobCleanupManager.__name__)` versus `job_models.JobModel.do_unfinished_jobs_exist(cron_services.JobModelsCleanupManager.__name__)` it basically checks if the job in question is not running.